include/aws/common/logging.h

```diff
@@ -58,8 +58,10 @@ enum aws_log_level {
 typedef uint32_t aws_log_subject_t;
 
 /* Each library gets space for 2^^10 log subject entries */
-#define AWS_LOG_SUBJECT_STRIDE_BITS 10
-#define AWS_LOG_SUBJECT_STRIDE (1U << AWS_LOG_SUBJECT_STRIDE_BITS)
+enum {
```

Comment:
Not a fan of this change.  Adjacent integral constant defines is not necessarily something that should be an enum

Reply:
is your concern mostly with just grouping of unrelated constants under the same enum or enum usage for constants in general?